doc/source/contributor/environments/aufn-ceph.rst

```diff
@@ -9,7 +9,7 @@ This environment creates a Universe-from-nothing_-style deployment of Kayobe con
 .. warning::
 
     This guide was written for the Yoga release and has not been validated for
-    Caracal. Proceed with caution.
+    Master. Proceed with caution.
```

Comment:
Maybe this should be validated at some point

Reply:
ditto for the other guides

Reply:
But that'll be another PR